dist_linux.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash -e
+
+template_dir="dist-template/windows"
```

Comment:
Can it be a typo - dist-template/windows? The same in dist_windows.sh - dist-template/linux.